src/service.ts

```diff
@@ -17,19 +15,22 @@ import { createIssueFromInternalError, Issue } from './issue';
 
 const rpc = new RpcProvider(message => {
   try {
-    process.send!(message, undefined, undefined, error => {
-      if (error) {
-        process.exit();
-      }
-    });
+    if (process.send) {
```

Comment:
Can this not exist?

Reply:
From the Node.js documentation: 

> If Node.js was not spawned with an IPC channel, process.send will be undefined.

I added this check to avoid the not-null assertion - maybe we should throw an Error instead of failing silently.

Reply:
Hmmmm.... Not sure. Maybe this is fine. We could `console.error` in the case of it being `undefined` but I'm not sure if there's value in that or not.